inlang/source-code/sdk/src/persistence/filelock/acquireFileLock.ts

```diff
@@ -0,0 +1,127 @@
+import { type NodeishFilesystem } from "@lix-js/fs"
+import type { NodeishStats } from "@lix-js/fs"
+import { debugLock } from "../../loadProject.js"
```

Comment:
Please fix as mentioned in https://github.com/opral/monorepo/pull/2709/#pullrequestreview-2042655176